api/app/models/theq/timezone.py

```diff
@@ -0,0 +1,30 @@
+'''Copyright 2018 Province of British Columbia
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.'''
+
+from qsystem import db
+from app.models.theq import Base
+
+
+class Timezone(Base):
+
+    timezone_id = db.Column(db.Integer, primary_key=True, autoincrement=True, nullable=False)
+    timezone_abbreviation = db.Column(db.String(3), nullable=False)
+    timezone_name = db.Column(db.String(256), nullable=False)
```

Comment:
How does this work with the moment libraries? I thought we planned on using `America/Vancouver`, `America/Dawson_Creek`, and `America/Edmonton` for the three timezones. If we use those, moment.js should be able to figure out the implied timezone automatically, and configure based around the clocks changing as they see fit. If we go with this, then they'll have to convert around the clock change themselves and update their office timezone twice yearly, yes?

Reply:
That's what this field was made for.

Reply:
Your test data of `MST` and `Mountain Standard Time` got me all confused